### PR TITLE
Worked on Issue #7551 Misspelling in documentation

### PR DESCRIPTION
**Pull request by Mitchkoens**
An attempt as Solving "Missprint in the documentation?" Issue #7551, Currently The documentation has a Few errors I think i found all of the changes needed.

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -387,6 +387,5 @@ harshil goel <harshil158@gmail.com>
 Lokesh Sharma <lokeshhsharma@gmail.com>
 Sartaj Singh <singhsartaj94@gmail.com>
 Chris Swierczewski <cswiercz@gmail.com>
-Sumith <sumith1896@gmail.com>
```
Comment (smichr) [inline AUTHORS]: this change should not be made

File: doc/src/aboutus.rst [hunk 2]
```diff
@@ -392,7 +392,6 @@ want to be mentioned here, so see our repository history for a full list).
 #. Lokesh Sharma: reorder setup.py file imports to correct NameError
 #. Sartaj Singh: use \left \| instead of \lvert for latex Abs
 #. Chris Swierczewski: RootOf.evalf watches for root on interval boundary
-#. Sumith: implemented columnspace of a matrix
```
Comment (smichr) [inline doc/src/aboutus.rst]: this change should not be made

File: sympy/core/expr.py [hunk 3]
```diff
@@ -682,6 +682,10 @@ def equals(self, other, failing_expression=False):
             return diff
         return None
 
+    def _eval_is_composite(self):
```
Comment (smichr) [inline sympy/core/expr.py]: this change should not be made

**Comment by Mitchkoens** [comment]
Sorry about that I accidentally merged with someone else's changes i just need to back to only the changes I made.

**Comment by Mitchkoens** [comment]
I think this should of fixed it back to what i first had, once again sorry about that still trying to get used to using git.

**Comment by certik** [comment]
@Mitchkoens you still need to remove that change that @smichr pointed out.

**Comment by certik** [comment]
Otherwise is the only change from `j` to `l`? I am fine with that, either way is fine with me.

**Comment by Mitchkoens** [comment]
@certik Sorry it had taken me so long to get back to that I have been having a very busy summer and but I have gotten around to making those changes now. The changes where in relation to Issue #7551 if I did not fix all of this issue or I misunderstood the issue can I ask for some guidance

File: sympy/physics/wigner.py [hunk 4]
```diff
@@ -344,8 +344,8 @@ def racah(aa, bb, cc, dd, ee, ff, prec=None):
 
     .. math::
 
-       Wigner6j(j_1,j_2,j_3,j_4,j_5,j_6)
-       =(-1)^{j_1+j_2+j_4+j_5} W(j_1,j_2,j_5,j_4,j_3,j_6)
+       Wigner6j(l_1,l_2,l_3,j_4,j_5,j_6)
```
Comment (leosartaj) [inline sympy/physics/wigner.py]: There is a mix-up of j and l here and some places more. See [this](https://github.com/sympy/sympy/issues/7551#issuecomment-45048133)

**Comment by certik** [comment]
@Mitchkoens thanks for getting back to it. I just want to clarify --- is the only change in this PR a change from `j` to `l`? I tried to see if anything else was done in the diff, and it seems that is it. Can you confirm it, or am I missing some more substantial change?

**Comment by Rufflewind** [comment]
I believe the convention is to have `j` (total angular momentum) be any integer or half-integer, whereas `l` (orbital angular momentum) can only be integer.  So it should be `j` for Wigner symbols, whereas for the Gaunt symbol it should be `l`.

**Comment by hargup** [comment]
The issue corresponding to this PR was fixed with https://github.com/sympy/sympy/pull/9752.

**Pull request closed without merging**